ci/benchmarks/partial-conv/esm2_pretrain.yaml

```diff
@@ -0,0 +1,54 @@
+scope: partial-conv
+time_limit: 14400
+script_args:
+  # All arguments referenced in the script string must be specified here.
+  # Arguments not referenced in the script string must have the 'arg' field specified.
+  # See jet/core/configs.py for the specification of the configuration class
+  workspace:
+    value: /workspace/bionemo2
+    key_segment: False
+  data_path:
+    value: /data/20240809_uniref_2024_03/data
+    key_segment: False
+  model:
+    value: esm2
+  variant:
+    value: train
+  config_name:
+    value: 650M
+  precision:
+    value: [bf16-mixed]
+  nodes:
+    value: [4]
+  gpus:
+    value: 8
+  batch_size:
+    value: 16
+  max_steps:
+    value: 26500
+script: |-
+  WANDB_API_KEY=$BIONEMO_WANDB_API_KEY ${variant}_${model} \
```

Comment:
Do you get anything with this `variant_model` thing? The following arguments are all specific to esm2, so maybe it would be clearer to just put `train_esm2` here and people see that it's just the command that's getting executed? There's no rule at the moment for how train scripts are named...

Reply:
model and variant are required in JET logs as identifiers in job names or logs. I know it is esm2 specific

Reply:
all listed parameters are used as identifiers